conversations.yaml

```diff
@@ -476,10 +522,11 @@ components:
         id:
           type: string
           description:
-            The PageId value configured in Conversations at the creation of the channel.
+            The PageId value configured in Medallia Conversations when 
+            the channel is created.
         time:
           type: integer
-          format: int64
+          format: int32
```

Comment:
Time should not be a 32-bit integer anymore.  https://en.wikipedia.org/wiki/Year_2038_problem

If Conversations is currently using a 32-bit representation, we should file a Jira ticket to upgrade to 64-bit and keep this as 64- bit.